src/parser/jobs/ast/modules/ArcanaGroups.ts

```diff
@@ -18,18 +24,18 @@ export const MINOR_ARCANA: Array<keyof ActionRoot> = [
 	'LADY_OF_CROWNS',
 ]
 
-export const SOLAR_SEAL_ARCANA: Array<keyof ActionRoot> = [
+export const DAMAGE_INCREASE_ARCANA: Array<keyof ActionRoot> = [
 	'THE_BALANCE',
-	'THE_BOLE',
+	'THE_SPEAR',
 ]
 
-export const LUNAR_SEAL_ARCANA: Array<keyof ActionRoot> = [
+export const HEAL_MIT_ARCANA: Array<keyof ActionRoot> = [
 	'THE_ARROW',
-	'THE_EWER',
+	'THE_BOLE',
 ]
 
-export const CELESTIAL_SEAL_ARCANA: Array<keyof ActionRoot> = [
-	'THE_SPEAR',
+export const REGEN_SHIELD_ARCANA: Array<keyof ActionRoot> = [
+	'THE_EWER',
```

Comment:
We can simply group these logically into DEFENSIVE_ARCANA too, as the way they are grouped now does not provide any logical benefit (to me at least...)

Reply:
updated!